components/button/button.css

```diff
@@ -0,0 +1,67 @@
+:root {
+    --btn-color: #059669;
```

Comment:
use meaningful names like --color-green, no need to mention btn here as this color can be used anywhere in the app. same goes for other variables.